<launch>
  <group ns="robot_0">
    <!-- Run the action servers -->
    <node pkg="tw11" type="ActionMove2Pos.py" name="ActionMove2Pos" 
          args="$(find mystage_ros)/world/AR/factory_laser_with_landmarks.world"
          required="true"/>
    <node pkg="tw11" type="ActionRecharge.py" name="ActionRecharge"
        required="true" output="screen"/>
    <node pkg="tw11" type="ActionRotate2Angle.py" name="ActionRotate2Angle"
        required="true" output="screen"/>
    <node pkg="tw11" type="ActionStop.py" name="ActionStop"
        required="true" output="screen"/>
  </group>
</launch>
